packages/bindings/src/gov.rs

```diff
@@ -86,6 +85,48 @@ pub enum GovProposal {
         /// all code ideas that should be removed from cache to free space
         code_ids: Vec<u64>,
     },
+    // ClientUpdateProposal is a governance proposal. If it passes, the substitute
```

Comment:
Doc comments? Also, a url reference to the relevant Cosmos SDK / tgrade sources section would be good.